homework-submission/week5-homework/src/stores/TodoStore.js

```diff
@@ -0,0 +1,148 @@
+import { configure, observable, action, computed, runInAction } from "mobx";
+import uuidv4 from "uuid/v4";
+const url = "https://hyf-react-api.herokuapp.com";
+
+configure({ enforceActions: "observed" });
+class TodoClass {
+  @observable
+  todos = [];
+  @observable
+  state = "Loading";
+
+  @action
+  async getTodos() {
+    this.state = "Loading";
+    this.todos = [];
+    try {
+      let todos = await this.getData();
+      runInAction(() => {
+        this.state = "Done";
+        this.todos = todos;
+      });
+    } catch (error) {
+      runInAction(() => {
+        this.state = "error";
+      });
+    }
+  }
+
+  @action
+  toggleDone = async id => {
```

Comment:
you have here two different kind of Code style: 
 1. line 13 `async getTodos() {/*Some Code*/}`
2. line 30 `toggleDone = async id => {/*Some Code*/}`

you have to relay on a one style of type so making your code more readable and understandable.

in the near future you will notice that there is a very big different between a function and an arrow function and I think you did already from your last homework as I saw 😃.